src/briefcase/platforms/iOS/xcode.py

```diff
@@ -370,10 +370,12 @@ def _install_app_requirements(
             pip_args=[
                 f"--platform=ios_{ios_min_tag}_arm64_iphoneos",
             ],
+            install_hint="It may also indicate that an `iphonesimulator` wheel could be found, "
+            "but an `iphoneos` wheel could not be found.",
             pip_kwargs={
                 "env": {
                     "PYTHONPATH": str(device_platform_site),
-                },
+                }
```

Comment:
As above, ordering should be consistent (and keep pip args/kwargs together) - unfortunately I can't put this in a code suggestion because there's a deleted line...